doc/modules/array_api.rst

```diff
@@ -24,6 +29,12 @@ explicitly as explained in the following.
     Currently, only `array-api-strict`, `cupy`, and `PyTorch` are known to work
     with scikit-learn's estimators.
 
+The following video provides an overview of the standard's design principles
+and how it facilitates interoperability between array libraries:
+
+- `Scikit-learn on GPUs with Array API <https://www.youtube.com/watch?v=c_s8tr1AizA>`_
+  by `Thomas J. Fan`_ at PyData NYC 2023.
```

Comment:
Fine with having the video, although at the same time it can be slightly outdated since array API is evolving.

I guess use `:user:` as in the changelog:
```suggestion
  by :user:`Thomas Fan <thomasjpfan>` at PyData NYC 2023.
```